npm/private/npm_translate_lock.bzl

```diff
@@ -135,12 +135,28 @@ _ATTRS = {
         doc = """If true, runs preinstall, install and postinstall lifecycle hooks on npm packages if they exist""",
         default = True,
     ),
+    "lifecycle_hooks_envs": attr.string_list_dict(
+        doc = """Environment variables applied to the preinstall, install and postinstall lifecycle hooks on npm packages.
+            Such as some package can define a PREBUILT_BINARY environment variable to download a os specific binary file
+            directly instead of build locally.
+            The environment variables can be defined per package or globally, you can define multiple
+            key/value pairs by providing an array of "key=value" entries.
+
+            For example:
+
+            lifecycle_hooks_envs: {
+	            # Use * to define environment variables for all packages
```

Comment:
I think the information in this comment should be up above when describing the API, not in a comment.

Reply:
Got it, I move the information up to the doc.